packages/modules/bigcommerce/src/services/bigcommerce-token-service.ts

```diff
@@ -0,0 +1,95 @@
+import { ExecutionContext, forwardRef, Inject, Injectable } from 'graphql-modules';
+
+import { CACHE_ITEMS_TTL, CACHE_KEY__CUSTOMER_IMPERSONATION_TOKEN } from '../constants';
+import { ModuleConfig } from '../providers';
+import {
+    BigCommerceModuleConfig,
+    createCustomerImpersonationToken,
+    getDataFromMeshCache,
+} from '..';
+import { getUnixTimeStampInSecondsForMidnightTonight } from '@aligent/utils';
+import { sign } from 'jsonwebtoken';
+
+// The default "customer_impersonation_token" query TTL should one not be stored in "CACHE_ITEMS_TTL"
+const QUERY_DEFAULT_TTL = CACHE_ITEMS_TTL?.[CACHE_KEY__CUSTOMER_IMPERSONATION_TOKEN] || 86400000;
```

Comment:
No. I think the fallback code is there because typescript suggests that this value might be undefined. This can be improved with `as const`, I'll see if I can do that and change other places we've done this in the code base.